web/index.html

```diff
@@ -67,6 +67,9 @@
           ga('send', 'event', 'user-input', 'success', line);
         }
 
+        const url = "/?q=" + encodeURIComponent(line)
```

Comment:
Will `const` really work for all (possibly mildly outdated) browsers? I think I would prefer a simple `var` here, even if `const` would be cleaner from a theoretical point of view.

Also, could you please add semicolons (for consistency)?